src/pages/blog/[...page].astro

```diff
@@ -2,15 +2,16 @@
 import type { Page } from "astro";
 import { createExcerpt } from '../../utils/create-excerpt';
 import BaseLayout from '../../layouts/BaseLayout.astro';
+import Pagination from '../../components/blog/Pagination.astro';
```

Comment:
Part of the conversion to the new Content Layer API, see https://docs.astro.build/en/guides/upgrade-to/v5/#updating-existing-collections and also implementation of the Pagination component